doc/library/tensor/signal/index.txt

```diff
@@ -20,5 +20,5 @@ forms of signal processing.
     :maxdepth: 1
 
     conv
-    downsample
     pool
+    downsample
```

Comment:
@nouiz I tought you wanted to keep it not to break the documentation. I can remove it if you want!